<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dllui</class>
 <widget class="QDialog" name="dllui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1368</width>
    <height>1081</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="buttonLogin">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>540</y>
     <width>161</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonID">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>410</y>
     <width>241</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>450</y>
     <width>75</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Cardnumber</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineCard">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>450</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>480</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>PIN</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="linePIN">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>480</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonCloseDLL">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>540</y>
     <width>64</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>510</y>
     <width>55</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Attempts</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineAttempts">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>510</y>
     <width>154</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="yhteysInfo">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>580</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
